functions/output_display.py

```diff
@@ -25,3 +29,123 @@ def deg(rad: float) -> float:
+        Args:
+            value (float): Value in decimal
+        Returns:
+            str: Value in binary
+    """
+    the_array = np.array([])
+    while value != 0 and (the_array.size <= 10):
+        new_value = float(value * 2)
+        number_to_store = int(new_value)
+        value = new_value % 1
+        the_array = np.append(the_array, int(number_to_store))
+    binary_string = "0."
+    for x in the_array:
+        binary_string = binary_string + str(int(x))
+    return binary_string
+
+
+def convert_to_binary(number: float) -> str:
+    """Returns the binary of a number
+        Args:
+            number (float): Value in decimal
+        Returns:
+            str: Value in binary
+    """
+    if number >= 0:
+        pos_sign = True
+    else:
+        pos_sign = False
+    if number % 1 == 0:  # Means that there isn't a fraction
+        binary_result = decimal_to_binary_integer(number)
+    else:  # Means that it's not a whole number and integer and fraction will be calculated individually
+        binary_result = decimal_to_binary_integer(int(number)) + decimal_to_binary_decimal(number % 1)
+    if pos_sign:
+        return binary_result
+    else:
+        return "-" + binary_result
+
+
+def convert_to_decimal(number: float) -> float:
+    """Returns a float that contains the equivalent of the binary value entered
+         Args:
+             number (float): Value in binary
+         Returns:
+             float: Value in decimal
+     """
+    if number >= 0:
+        pos_sign = True
+    else:
+        pos_sign = False
+    if number % 1 == 0:  # Means that there isn't a fraction
+        decimal_result = int_binary_to_decimal(number)
+    else:  # Means that it's not a whole number and integer and fraction will be calculated individually
+        string_number = str(number)
+        decimal_index = string_number.index('.')
+        decimal_number = float(string_number[decimal_index:])
+        decimal_result = int_binary_to_decimal(int(number)) + float_binary_to_decimal(decimal_number % 1)
+    if pos_sign:
+        return decimal_result
+    else:
+        return decimal_result * -1
+
+def int_binary_to_decimal(value: float) -> float:
+    """Returns a float that contains the integer equivalent of the binary value entered
+        Args:
+            value (float): Value in binary
+        Returns:
+            float: Value in decimal
+    """
+    the_number = str(value)
+    position = 0
+    value = 0
+    for char in reversed(the_number):
+        # print(int(char))
+        value = value + (int(char) * exponents.pow(2,position))
+        position = position + 1
+    return value
+
+
+def float_binary_to_decimal(value: float) -> float:
+    """Returns a float that contains the decimal equivalent of the binary value entered
+        Args:
+            value (float): Value in binary
+        Returns:
+            float: Value in decimal
+    """
```

Comment:
This could maybe do with error checking to make sure that the input is within some acceptable range? I don't think that we will run into any problems since the code is nearing finalization and we won't have to maintain it much further beyond that but on a long term project I'd include a check like that in case somebody uses the function wrong.

Add it if you think that it's necessary.

Reply:
I'm thinking that since we won't run into any issues for our purposes that it'll probably be okay for now